### DISCUSSION TITLE
App Integrity threat not detected in iOS device || Expo React Native

### DISCUSSION THREAD

**Discussion by Mr-harsh10** [talsec/Free-RASP-ReactNative#66] [Q&A]
I am working on a React Native Expo app. During penetration testing, the tester was able to tamper with my app, rebundle it, and run it without any issues. I need to prevent this.

Dependencies
"expo": "^49.0.0",
"react-native": "0.72.10",
"freerasp-react-native": "^3.7.2",

**Comment by tompsota** [comment]
Closing the discussion as it is duplicate of #68
